src/Spago/Config.purs

```diff
@@ -149,8 +148,8 @@ data Package
 
 -- | Reads all the configurations in the tree and builds up the Map of local
 -- | packages to be integrated in the package set
-readWorkspace :: forall a. Maybe PackageName -> Spago (Git.GitEnv a) Workspace
-readWorkspace maybeSelectedPackage = do
+readWorkspace :: Maybe PackageName -> Boolean -> Spago (Registry.RegistryEnv _) Workspace
```

Comment:
I'm always a little leery of booleans as arguments; what do you think of either making this a record:

```purs
{ selectedPackage :: Maybe PackageName
, pureBuild :: Boolean
}
```

or a data type

```purs
data PureBuild = NotPure | Pure
```

Reply:
Sure, either works